src/client/components/SuggestedRoutes/SuggestedRouteSummary.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import TitleAndDescribe from "../TitleAndDescribe";
+
+export default class SuggestedRouteSummary extends Component {
+  render() {
+    const { id, isSelected, handleClick, time, price } = this.props;
+    return (
+      <div>
+        {isSelected ? <h4>SELECTED ROUTE</h4> : null}
+        <TitleAndDescribe title="HOURS" description={time} />
+        <TitleAndDescribe title="TOTAL COST" description={price} />
+        {!isSelected ? (
+          <button onClick={() => handleClick(id)}>SELECT ROUTE</button>
```

Comment:
this is not wrong, but a bad practice/habit, always try to strip out functions out of the render scope if you can.
```
const onSelectRoutClick = () => {
    this.props.handleClick(this.props.id)
}
  render() {
    const { id, isSelected, handleClick, time, price } = this.props;
    return (
      <div>
        {isSelected ? <h4>SELECTED ROUTE</h4> : null}
        <TitleAndDescribe title="HOURS" description={time} />
        <TitleAndDescribe title="TOTAL COST" description={price} />
        {!isSelected ? (
          <button onClick={onSelectRoutClick}>SELECT ROUTE</button>
```
the issue with the way you did it, is that on every render you create a new whole function, when you strip the arrow function out like above, it get created once on the fist load and it just keep referring to it on each render after that, which is good for performance.